python/gui/auto_generated/qgsquerybuilder.sip.in

```diff
@@ -114,8 +114,6 @@ Load query from the XML file
 .. versionadded:: 3.16
 %End
 
-    void setDatasourceDescription( const QString &uri );
```

Comment:
This is an API break -- does it need to be removed?

Reply:
oups... Afaict, this is used to fill the "set provider filter on..." text at the top and I removed that QLabel widget, putting the (default) info in the window title. To avoid redundancy.
Should I add back the widget (emptied by default?), move the function to alter the window title (weird!) or consider that there is a low risk to break anything (not clear to me what info people would usually put there)?